code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -35,192 +35,306 @@
 	id = "nicotine"
 	description = "Slightly reduces stun times. If overdosed it will deal toxin and oxygen damage."
 	reagent_state = LIQUID
-	color = "#60A584" // rgb: 96, 165, 132
-	addiction_threshold = 30
+	color = rgb(0,0,0)
+	overdose_threshold = 30
 
 /datum/reagent/drug/nicotine/on_mob_life(mob/living/M)
 	if(prob(1))
 		var/smoke_message = pick("You feel relaxed.", "You feel calmed.","You feel alert.","You feel rugged.")
 		M << "<span class='notice'>[smoke_message]</span>"
-	M.AdjustStunned(-1, 0)
+	if(prob(50))
+		M.AdjustStunned(-1, 0)
+		M.AdjustParalysis(-1, 0)
+		M.AdjustWeakened(-1, 0)
 	M.adjustStaminaLoss(-0.5*REM, 0)
 	..()
 	. = 1
 
+/datum/reagent/drug/nicotine/overdose_process(mob/living/M, severity)
+	var/effect = ..(M, severity)
```

Comment:
correct me if I"m wrong, but isn't it unnecessary to pass the arguments there, when both are included, by default in all `overdose_process`?